client/src/components/timetable/ExpandedEventView.tsx

```diff
@@ -193,6 +193,10 @@ const ExpandedEventView: React.FC<ExpandedEventViewProps> = ({
     delete updatedEventData[id];
     setCreatedEvents(updatedEventData);
   };
+
+  // link sharing url for custom events
+  const url = 'localhost:3000/event/' + btoa(JSON.stringify(eventPeriod));
```

Comment:
will this work with the hardcoded 'localhost:3000' url when we deploy it?

Reply:
I think you'll have to get the current URL of the webpage

Reply:
should be able to do something like: 
```suggestion
const currentUrl = window.location.href;
const url = currentUrl + 'event/' + btoa(JSON.stringify(eventPeriod));
```